README.md

```diff
@@ -59,7 +59,7 @@ Contributing
 ------------
 If you would like to make a change to the site:
 
-1. Fork the [github mirror](https://github.com/apache/apex-site)
+1. Fork the [github repository](https://github.com/apache/apex-site)
```

Comment:
Fork the github [Apex site repository](https://github.com/apache/apex-site) to your github account.